src/storyMap/components/StoryMapsHomeCard.js

```diff
@@ -51,24 +49,23 @@ const List = withProps(BaseList, {
 
 const GridListItem = withProps(Grid, { component: 'li' });
 
-const StoryMapsHomeCard = props => {
-  const dispatch = useDispatch();
+const StoryMapsHomeCard = ({
```

Comment:
Issue: I think since we are not just using this in the home page anymore, I think we shoudl rename it to `StoryMapsCard`